ckanext/stadtzhtheme/logic.py

```diff
@@ -1,42 +1,63 @@
 import pysolr
-import re
-from unidecode import unidecode
 from ckan.plugins.toolkit import get_or_bust, side_effect_free
-from ckan.logic import ActionError, NotFound, ValidationError
+from ckan.logic import ActionError
 from ckan.lib.search.common import make_connection
+import ckan.plugins.toolkit as tk
 
 import logging
 log = logging.getLogger(__name__)
 
 
 @side_effect_free
 def ogdzh_autosuggest(context, data_dict):
+    """
+    collecting autosuggestions from solr by calling
+    :param q: the search term
+           example: velo
+    :param cfq: a context as facet-names concatenated by AND
+           example: 'soziales AND bevolkerung'
+    :return: autosuggestions that were generated by solr
+             by the suggest handler
+             as a list of unique suggestions
+    """
     q = get_or_bust(data_dict, 'q')
     cfq = data_dict.get('cfq', '')
 
     if cfq:
         cfq = 'active AND %s' % cfq
     else:
         cfq = 'active'
-
     handler = '/suggest'
     suggester = 'default'
+    suggest_search_limit = int(tk.config.get(
+        'ckanext.stadtzh-theme.ogdzh_autosuggest_search_limit', 100))
+    suggest_results_limit = int(tk.config.get(
+        'ckanext.stadtzh-theme.ogdzh_autosuggest_result_limit', 10))
```

Comment:
can you add those config-values to our .ini-files in the root-repo, too? I think that makes it more clear and quicker to find out what values are used.